### ISSUE TITLE
Borders: ability to configure a border color around windows when using title bars, without a line inbetween

### ISSUE THREAD

**Issue by RicArch97** [WillPower3309/swayfx#70]
Currently, it is not possible in Sway to add a singular border around the entire view when using title bars without getting an extra line between the title bar and application window.
Sway provides the following config options: (man 5 sway)
```
border
               The border around the title bar.
child_border
               The border around the view itself.
```
When using this, the result ends up like this, and as you can see by the arrow pointing at it, there is now also a line between the border and the application view. I personally really don't want this, and it would be nicer to have a singular border around the entire window (application + title bar).

![2022-12-11T23:13:58,593641661+01:00](https://user-images.githubusercontent.com/61013287/206932309-f06a349f-1c35-4746-a98c-2bed14b61b86.png)

I think we should think of a way to provide a config option to do this, without breaking Sway's original functionality. Maybe with a double border? When setting a double border, and disabling Sway's original borders, we would get this result, plus would be nice for the users who want double borders anyways

**Comment by WillPower3309** [comment]
I agree with this, it seems like one rice managed to get the color to match: https://www.reddit.com/r/unixporn/comments/xxu2pd/sway_hopped_on_the_nixos_train/

**Comment by WillPower3309** [comment]
Would be simple to have a config option to have the bottom line match the titlebar color, may have to think of naming of that option though

**Comment by RicArch97** [comment]
> I agree with this, it seems like one rice managed to get the color to match: https://www.reddit.com/r/unixporn/comments/xxu2pd/sway_hopped_on_the_nixos_train/, although this could end up giving up borders in general, haven't looked at their specific config yet

This one doesn't use border **colors** though? The titlebar is colored, but that is done with the `background` option, i just left mine black because i think it looks nice. The borders are the grey lines you see in my screenshot above. To compare with and without border colors:
![2022-12-11T23:13:58,593641661+01:00](https://user-images.githubusercontent.com/61013287/206934067-05e17fed-38f0-40b1-8dd0-01b6921485fc.png)
![2022-12-11T23:57:32,903436462+01:00](https://user-images.githubusercontent.com/61013287/206934070-9e1d025b-642f-4d32-bc29-bb9cbed1a138.png)

Basically i'd want the grey line around the view (and yes thats when using tiltlebars) but not the extra grey line between the titlebar and terminal window that the arrow is pointing to

**Comment by WillPower3309** [comment]
Gotcha, I'll look into this once I've finished shadows. Should be pretty simple
